extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -134,3 +134,16 @@ export function doubleclickIsA4AEnabled(win, element) {
   }
   return enableA4A;
 }
+
+export const adConfig = {
```

Comment:
for fast fetch fall back case, we still skip renderstart: https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js#L237

clientIdScope is used by ad-cid.js .